inst/tinytest/test_fparse_fload.R

```diff
@@ -33,49 +33,49 @@ expect_error(fload(c(test_file1, test_file2)))
 
 #* valid -----------------------------------------------------------------------
 #** fparse() -------------------------------------------------------------------
-expect_identical(fparse(NA_character_), NA)
-expect_identical(fparse("1"),
-                 1L)
-expect_identical(fparse(c("1", "2")),
-                 list(1L, 2L))
-expect_identical(fparse(c(json1 = "1", json2 = "2")),
-                 list(json1 = 1L, json2 = 2L))
-expect_identical(fparse(NA_character_),
+expect_equal(fparse(NA_character_), NA)
```

Comment:
Tests for equality in R are not the same as tests for identity.

If the library behaviour changed we can discuss updating it.  Lacking motivation for _why_ this is being changed I am a little sceptical.